### PR TITLE
Fix/RO-549/problem importing rdf

### PR DESCRIPTION
**Pull request by luisfn**
Ticket: [RO-549](https://oat-sa.atlassian.net/browse/RO-549)

The goal of this PR is to create the missing config file during the extension install. This file is needed to define the strategy to be used during RDF import process

### CODE REVIEW

**Review by Skyld — APPROVED** [review]

File: config/default/rdfImporterConfig.conf.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return new oat\oatbox\config\ConfigurationService(array(
```
Comment (gabrielfs7) [inline config/default/rdfImporterConfig.conf.php:3]: I saw we already had a migration for this, but we forgot this in the installation: 

https://github.com/oat-sa/extension-tao-testtaker/blob/master/migrations/Version202105041023253148_taoTestTaker.php#L29

@mike-ionut-mihai-sandu-tao

Comment (luisfn) [reply config/default/rdfImporterConfig.conf.php:3]: Yes, just rechecked, if we have the default file, it's is copied

File: config/default/rdfImporterConfig.conf.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    'strategy' => 'fail'
```
Comment (gabrielfs7) [inline config/default/rdfImporterConfig.conf.php:4]: I would use the constants here instead:

```suggestion
    RdfImporter::OPTION_STRATEGY => RdfImporter::OPTION_STRATEGY_FAIL_ON_DUPLICATE
```

**Review by gabrielfs7 — APPROVED** [review]

**Review by bartlomiejmarszal — COMMENTED** [review]
Is this connected with any other PR? To give this PR more context?

**Review by Skyld — APPROVED** [review]

**Comment by luisfn** [comment]
> Is this connected with any other PR? To give this PR more context?

No other PR related directly. Creation of this file was missed in the past, so new installs are not creating this config.

You can see more details on the Ticket RO-549 (In the description)

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**